tests/puppeteer/.docker/Dockerfile

```diff
@@ -3,23 +3,23 @@
 # Based upon:
 # https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#running-puppeteer-in-docker
 
-FROM node:8-slim
+FROM node:10-slim
 
-RUN  apt-get update \
-     # See https://crbug.com/795759
-     && apt-get install -yq libgconf-2-4 \
-     # Install latest chrome dev package, which installs the necessary libs to
-     # make the bundled version of Chromium that Puppeteer installs work.
-     && apt-get install -y wget --no-install-recommends \
-     && wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+# Install latest chrome dev package and fonts
+RUN  wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
      && sh -c 'echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list' \
      && apt-get update \
-     && apt-get install -y google-chrome-unstable --no-install-recommends \
+     && apt-get install -y google-chrome-unstable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-kacst \
```

Comment:
Just thinking, is it a good idea to use `google-chrome-unstable`?
Should be `google-chrome-stable`

Reply:
`google-chrome-unstable` is used by developers of puppeteer. why? I don't really know.
Maybe to test sooner the beta version of chrome with puppeteer.
Should we do the same thing ? Not necessarily but I prefer to have my DockerFile based on the one in the official page.

Reply:
Watch out, reading their FAQ, it's only because they contribute to Chromium and also specific to puppeteer development cycle.

If you do this, you must update `puppeteer` as often as they do.
Each version of puppeteer target a specifc version of chromium: https://github.com/GoogleChrome/puppeteer/blob/master/package.json#L11

Reply:
ok, changes done.
thanks